index.html

```diff
@@ -264,8 +264,8 @@
         </a>
       </div>
       <div class="col-md-6">
-        <p>ClojureBridge Berlin is organized with a lot of love by volunteers. We try our best to make the workshop a memorable event and bring more women* into programming.</p>
-        <p><i>ClojureBridge Berlin wird mit ganz viel Liebe von Freiwilligen organisiert. Wir versuchen unser Bestes, aus dem Workshop ein unvergessliches Event zu machen und mehr Menschen(FLTI) zum Programmieren anzuregen.</i></p>
+        <p>ClojureBridge Berlin is organized with a lot of love by volunteers. We try our best to make the workshop a memorable event and bring more people into programming.</p>
```

Comment:
I considered that, but I think we stress that enough in other places. This paragraph is primarily about the fact that it's organised by volunteers & with a lot of love. Do you agree / does that make sense? 
(compare the German version: "menschen zum programmieren anregen")